### PR TITLE
Starts Python's SimpleHttpServer without cache.

### PR DESCRIPTION
**Pull request by hoangviet1993**
This PR addresses #2 's gotcha :)

Code was taken from this [SO](https://stackoverflow.com/questions/12193803/invoke-python-simplehttpserver-from-command-line-with-no-cache-option) post.

### CODE REVIEW

File: scripts/serve.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
```
Comment (seanlip) [inline scripts/serve.py]: Add license header, as well as the source SO link.

File: scripts/serve.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import SimpleHTTPServer
+
+class MyHTTPRequestHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
```
Comment (seanlip) [inline scripts/serve.py]: Name this DevServerHttpRequestHandler or similar

File: scripts/serve.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    def end_headers(self):
+        self.send_my_headers()
+        SimpleHTTPServer.SimpleHTTPRequestHandler.end_headers(self)
+
+    def send_my_headers(self):
```
Comment (seanlip) [inline scripts/serve.py]: Use a clearer name -- send_no_cache_headers maybe. Or why not just inline this under end_headers()?

File: scripts/serve.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+        self.send_header("Cache-Control", "no-cache, no-store, must-revalidate")
```
Comment (seanlip) [inline scripts/serve.py]: Lint: typically we use single-quotes throughout. Ditto below.

File: scripts/serve.py (hunk 1 above)
Comment (hoangviet1993) [reply scripts/serve.py]: Done!

File: scripts/serve.py (hunk 3 above)
Comment (hoangviet1993) [reply scripts/serve.py]: `end_headers()` now has `send_my_headers()` contents. Removed `send_my_headers()`!

File: scripts/serve.py (hunk 4 above)
Comment (hoangviet1993) [reply scripts/serve.py]: Fixed!

**Comment by hoangviet1993** [comment]
All comments addressed, PTAL @seanlip 👍

**Pull request merged**